resources/responses/skroutz/merchants_ecommerce_orders_demo_rejected.yml

```diff
@@ -85,6 +85,10 @@
     "created_at": "2021-06-24T13:11:58+03:00",
     "expires_at": "2021-06-25T13:11:58+03:00",
     "dispatch_until": "2021-06-28T13:11:58+03:00",
-    "express": false
+    "express": false,
+    "rejection_info": {
+      "reason": "Our store is closed for personal reasons",
```

Comment:
Let's use Greek to be in sync with the rest of the fields.

Reply:
Done